app/Http/Controllers/Auth/AuthController.php

```diff
@@ -31,13 +31,20 @@ class AuthController extends Controller
     protected $redirectTo = '/';
 
     /**
-     * Create a new authentication controller instance.
+     * The authentication guard.
      *
-     * @return void
+     * @var Guard
+     */
+    protected $guard;
+
+    /**
+     * Create a new authentication controller instance.
+     * @param Guard $guard
      */
-    public function __construct()
+    public function __construct(Guard $guard)
     {
-        $this->middleware($this->guestMiddleware(), ['except' => 'logout']);
+        $this->guard = $guard;
+        $this->middleware('role:admin', ['except' => 'logout']);
```

Comment:
This probably doesn't work, right? Since it'd be gating the `login` action to only allow logged-in admin users.